tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -128,6 +140,10 @@ def rdf_to_json(file_path, class_list):
     return json_schema_list
 
 
+def write_to_json(out_file_name,json_dict,key):
+    with open(out_file_name, 'w') as f:
+        logging.info(json.dumps(json_dict[key], indent=4))
+        json.dump(json_dict[key], f)
```

Comment:
⛏️ Add a space below for formatting. (I've dropped [DSPDC-1676](https://broadinstitute.atlassian.net/browse/DSPDC-1676) to get auto formatting setup, another small bite).